region/tourian/main.json

```diff
@@ -3876,13 +3879,43 @@
                   "notable": false,
                   "requires": [
                     {"or": [
-                      "SpaceJump",
-                      "canConsecutiveWalljump"
+                      "canCarefulJump",
+                      "ScrewAttack",
+                      "Morph",
+                      {"enemyDamage": {
+                        "enemy": "Tourian Space Pirate (all)",
+                        "type": "contact",
+                        "hits": 1
+                      }}
+                    ]},
+                    {"or": [
+                      "canCarefulJump",
+                      "ScrewAttack",
+                      "HiJump"
+                    ]},
+                    {"or": [
+                      "canConsecutiveWalljump",
+                      {"and": [
+                        "SpaceJump",
+                        {"or": [
+                          "HiJump",
+                          {"and": [
+                            "canCarefulJump",
+                            {"acidFrames": 20},
+                            {"enemyDamage": {
+                              "enemy": "Tourian Space Pirate (all)",
+                              "type": "contact",
+                              "hits": 1
+                            }}
+                          ]},
+                          {"acidFrames": 100}
+                        ]}
+                      ]}
                     ]}
                   ]
                 }
               ],
-              "note": "The acid starts rising when Samus is at the bottom of the long climb"
+              "note": "The acid starts rising when Samus is at the bottom of the long climb."
```

Comment:
A practical note on baiting the right side pirates to shoot right before you go up main shaft would be nice, but logically you can just kill them 🤔.
And a strat for falling through left pirates depends somewhat on how fast and on which side you climb main shaft.